envoy/api/v2/auth/local_rbac.proto

```diff
@@ -0,0 +1,64 @@
+syntax = "proto3";
+
+// [#proto-status: draft]
```

Comment:
I think we can remove draft stuff as long as we go with v2alpha (same elsewhere). Doesn't really matter though as long as we do v2alpha.